src/authentication/index.ts

```diff
@@ -35,15 +34,14 @@ export class PasskeyAuthenticationFlow {
   }
 
   async registrationOptions(username: string) {
-    const newId = uuid();
-    await this.userRepository.create({ id: newId, username });
+    await this.userRepository.create({ username });
     // const userAuthenticators =
     //   await this.authenticatorRepository.listByUserId(newId);
 
     const options = await generateRegistrationOptions({
       rpName,
       rpID,
-      userID: newId,
+      userID: username,
       userName: username,
```

Comment:
Is this necessary to have duplicated the `userID` and `userName` with the same value?

Reply:
@agustina-vighetti 90% sure. This function interfaces with WebAuthN and I am trying to make it fit with what we have.